nbi/buildspec.yml

```diff
@@ -19,16 +19,17 @@ phases:
     commands:
       - echo "Setting up conda"
       - export PATH=$LCC_CONDA_INSTALL_DIR/bin:$PATH
-      - for CHANNEL in conda-forge cf-staging; do conda config --system --append channels $CHANNEL; done
-      - conda install --force-reinstall --yes conda-pack=0.7.0
+      - conda install conda-libmamba-solver
+      - conda config --set solver libmamba
+      - conda install -c conda-forge conda-pack=0.7.0
```

Comment:
1. I believe you are planning to change this our codepipeline infra too? 
2. Is there a reason we have to route to libmamba for this, is this due to threads not available in the conda? 
3. If planning to make this change in codepipeline infra as well, have you tested this for NBI creation and whether notebooks are running fine?

Reply:
1) I am not sure which infra you mean here.
2) [libmamba](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) is a faster dependency management solver. This is not a thread constraint so much as replacing the default solver with a more performant and reliable one.
3) I have tested the `.tar` file but I have not tested any infrastructure changes.